shell/browser/api/electron_api_menu_mac.mm

```diff
@@ -54,6 +73,29 @@
   base::SequencedTaskRunnerHandle::Get()->PostTask(FROM_HERE, std::move(popup));
 }
 
+v8::Local<v8::Value> Menu::GetUserAcceleratorAt(int command_id) const {
+  v8::Isolate* isolate = JavascriptEnvironment::GetIsolate();
+  if (![NSMenuItem usesUserKeyEquivalents])
+    return v8::Null(isolate);
+
+  auto controller = base::scoped_nsobject<ElectronMenuController>(
+      [[ElectronMenuController alloc] initWithModel:model()
+                              useDefaultAccelerator:NO]);
+
+  base::scoped_nsobject<NSMenuItem> item =
+      [controller makeMenuItemForIndex:GetIndexOfCommandId(command_id)
+                             fromModel:model()];
+  if ([[item userKeyEquivalent] length] == 0)
+    return v8::Null(isolate);
+
+  NSString* user_key_equivalent = [item keyEquivalent];
```

Comment:
I'm not very familiar with these APIs, but it seems odd that you check `userKeyEquivalent` above, but then read the value from `keyEquivalent` and assign it to a variable named `user_key_equivalent`. Is this line also supposed to call `userKeyEquivalent`?

Reply:
You're correct that this is odd, but it is very much correct 😄 

`userKeyEquivalent` is a `NSString*` that contains either 0 or 1 character mapping to the literal key (ignoring modifiers) that the user has specified as the override.  If it's empty, there is no override.  The actual `keyEquivalent` and `keyEquivalentModifierMask` that the user specified are available in the standard format from the two keys we read from here.

It's weird but that's how Apple designed it 😆